README.md

```diff
@@ -73,79 +79,67 @@ privileges to perform those operations.
 ## Setup
 
 Instance-level configuration and keys are stored in the instance
-folder. When deploying this folder will need to be created manually.
+folder. There are two files in the instance folder. The first at
+`instance/config.py.sample.dev` is a sample config file suitable for
+development. The second is a file (`instance/env_to_config_shim.py`)
+that allows the use of environment variables for app configuration,
+which is intended to smooth production deployments.
 
-### Development setup
+### Development configuration
 
 To set up the app for local development you need to follow these steps:
 
-1.  Create a new instance directory:
+1.  Copy the configuration template file to a config file: 
     ```sh
-    mkdir instance
-    ```
-
-2.  Create a new file at `instance/config.py` and place this
-    configuration into it: 
-    ```python
-    SECRET_KEY = "Something super secret"
-    TEMPLATES_AUTO_RELOAD = True
-    ORDNANCE_SURVEY_PLACES_API_KEY = 
-    PERMANENT_SESSION_LIFETIME = 30 * 60
-    GA_TRACKING_ID =
-
-    # SENTRY CONFIG
-    SENTRY_DSN =
-
-    # NHS OIDC config
-    NHS_OIDC_AUTHORITY_URL = "https://auth.sandpit.signin.nhs.uk"
-    NHS_OIDC_CLIENT_ID =
-    NHS_OIDC_REGISTRATION_CALLBACK_URL = "http://localhost:5000/nhs-registration-callback"
-    NHS_OIDC_LOGIN_CALLBACK_URL = "http://localhost:5000/nhs-login-callback"
-    NHS_OIDC_SCOPES=["openid", "profile", "email", "phone", "profile_extended"]
-    NHS_OIDC_VTR=["P0.Cp.Cd", "P0.Cp.Ck", "P0.Cm"]
+    cp instance/config.py.sample.dev instance/config.py
     ```
-    Obviously, change the example hostnames as appropriate for your
-    setup.
 
-3.  Fill in the missing configuration values as per the **Configuration
+2.  Fill in the missing configuration values as per the **Configuration
     Variables Guide** section, and set the AWS environment variables as
     per the ***Environment Variables Guide***.
 
-4.  Place the private key file for your NHS oidc client id into the
+3.  Place the private key file for your NHS oidc client id into the
     path `instance/private_key.pem`.
 
-5.  Set the `FLASK_CONFIG_FILE` environment variable so that it points at
+4.  Set the `FLASK_CONFIG_FILE` environment variable so that it points at
     the new file. (Note: the root path for Flask config is the `instance`
     folder.) I.e.:
     ```sh
     export FLASK_CONFIG_FILE='config.py'
     ```
     
-6.  Set the Flask environment to development, and let Flask know where
+5.  Set the Flask environment to development, and let Flask know where
     the app file is stored:
     ```sh
     export FLASK_ENV='development'
     export FLASK_APP='run.py'
     ```
-
-7.  Run the build script that downloads the government dependencies:
+	
+6.  Run the app:
     ```sh
-    bash build.sh
+    flask run
     ```
+		
+### Production configuration
```

Comment:
Worth having some mention of the Concourse deployment in the README (apologies if I've missed it). I.e. the fact that config exists for it to deploy using Concourse, perhaps a simple explanation of the pipeline, and the fact that certain variables are pulled from Concourse ?

Reply:
Updated readme to reflect changes made